CONTRIBUTING.md

```diff
@@ -99,26 +100,29 @@ The conda organization adheres to the [NumFOCUS Code of Conduct](https://www.num
    $ docker compose run unit-tests
    ```
 
-   This will download the `conda-ci` Docker image from the
-   [Github Container Registry](https://github.com/conda/conda/pkgs/container/conda-ci)
-   and start `bash` with the conda development mode already enabled.
-   By default, it will pick the Python 3.9 image built from `master`. If you need
-   a different Python version, set a `CONDA_DOCKER_PYTHON` environment variable like this:
+   This builds the same Docker image as used in continuous
+   integration from the [Github Container Registry](https://github.com/conda/conda/pkgs/container/conda-ci)
+   and starts `bash` with the conda development mode already enabled.
+   By default, it will pick the Python 3.9 image built from `master`.
+
+   If you need a different Python version, set a `CONDA_DOCKER_PYTHON`
+   environment variable like this to rebuild the image:
 
    **Bash (macOS, Linux, Windows)**
 
    ```bash
-   $ CONDA_DOCKER_PYTHON=3.8 docker compose run unit-tests
+   $ CONDA_DOCKER_PYTHON=3.8 docker compose build unit-tests
```

Comment:
This is needed since just running the container with the env var won't trigger the rebuild.

Reply:
Will `build` also _run_ the service command after?

Reply:
I'll merge and we can continue the discussion in the main PR.